Tests/WebSocketKitTests/WebSocketKitTests.swift

```diff
@@ -262,6 +263,32 @@ final class WebSocketKitTests: XCTestCase {
         print("Waiting for server close...")
         try server.close(mode: .all).wait()
     }
+    
+    func testIPWithTLS() throws {
+        let port = 1337
```

Comment:
One way to get around this is by setting the port to 0. It will then pick a free port and you'll avoid issues of tests trying to use ports that are already in use https://github.com/vapor/vapor/blob/main/Tests/VaporTests/ApplicationTests.swift#L111-L139

Reply:
That works great, thanks!